src/languages/ruby.js

```diff
@@ -326,20 +375,28 @@ export default function(hljs) {
     METHOD_DEFINITION,
     {
       // swallow namespace qualifiers before symbols
-      begin: hljs.IDENT_RE + '::' },
+      begin: hljs.IDENT_RE + '::'
+    },
     {
       className: 'symbol',
       begin: hljs.UNDERSCORE_IDENT_RE + '(!|\\?)?:',
       relevance: 0
     },
     {
       className: 'symbol',
-      begin: ':(?!\\s)',
+      begin: '(?<!:):(?!\\s|:)',
```

Comment:
We can't use negative lookbehind until v12 as that would be a breaking change.

Reply:
The rule on line 378 is designed to deal with `::` and prevent false symbol positives - until v12 where it would be simpler to use negative lookbehind.

Reply:
Removing this negative lookbehind will cause this:
<img width="424" alt="image" src="https://github.com/user-attachments/assets/7bd5bf9b-7f97-412f-b162-68a3277a52b3" />

The rule on line 378 doesn't seem to do its job.

So I had to adjust line 378 so I can remove the negative lookbehind.